billy-andorra/src/main/java/com/premiumminds/billy/andorra/AndorraPersistenceDependencyModule.java

```diff
@@ -33,7 +34,8 @@ public AndorraPersistenceDependencyModule(String persistenceUnitId) {
 
     @Override
     protected void configure() {
-        JpaPersistModule persistModule = new JpaPersistModule(this.persistenceUnitId);
+        final var options = JpaPersistOptions.builder().setAutoBeginWorkOnEntityManagerCreation(true).build();
```

Comment:
https://github.com/google/guice/wiki/Guice600#persist
> Persist had a number of changes, some of which are backwards incompatible. Notably: injection of EntityManager no longer implicitly starts a unit of work (because this led to leaks). Users can opt-in to the legacy behavior by constructing the [JpaPersistModule](https://google.github.io/guice/api-docs/6.0.0/javadoc/com/google/inject/persist/jpa/JpaPersistModule.html#%3Cinit%3E(java.lang.String,com.google.inject.persist.jpa.JpaPersistOptions)) with a [JpaPersistOptions](https://google.github.io/guice/api-docs/6.0.0/javadoc/com/google/inject/persist/jpa/JpaPersistOptions.html) that sets [setAutoBeginWorkOnEntityManagerCreation](https://google.github.io/guice/api-docs/6.0.0/javadoc/com/google/inject/persist/jpa/JpaPersistOptions.Builder.html#setAutoBeginWorkOnEntityManagerCreation(boolean)) to true.

I was trying to keep the same behavior and only perform a javax -> jakarta change.